test/package-json-test.js

```diff
@@ -99,15 +99,50 @@ describe('generating package.json', function() {
       });
     });
 
-    it('contains additional config from addons', function() {
+    it('contains additional config from ember-fastboot-build-example addon', function() {
       let pkg = fs.readJsonSync(app.filePath('dist/package.json'));
 
       expect(pkg.fastboot.config['foo']).to.equal('bar');
     });
 
+    describe('with addon that implements fastbootConfigTree', function() {
```

Comment:
Would recommend moving this after the `'contains app name` test so it is easier to read.

Reply:
@kratiahuja I've moved `'contains app name'` up to make tests for application config grouped.
Thanks for feedback!